docs/woocommerce.com/analytics-basics.md

```diff
@@ -1 +1,92 @@
-# Using WooCommerce Analytics
\ No newline at end of file
+# Using WooCommerce Analytics Reports
+Most of the new reports in the WooCommerce Analytics section have a set of common tools. This document will introduce you to those features, and how they can be utlized to customize and dive-deeper into the various new analytics reports.
+
+## Date Range Picker
+![Analytics Date Range Picker](images/analytics-basics-date-range-picker.png)
+The Date Range picker allows you to specify which dates you want to include in the report being viewed. When you first open the Date Range Picker, a variety of popular _presets_ allow you to quickly chose some common date ranges:
+
+![Analytics Date Range Picker Presets](images/analytics-date-range-picker-presets.png)
+
+- Today
+- Yesterday
+- Week to Date
+- Last Week
+- Month to Date
+- Last Month
+- Quarter to Date
+- Last Year
+
+You can also select which date range to compare against, either the same range from the _Prior Year_ or the _Prior Period_.
+
+If the presets don't match your desired date range, click the _Custom_ tab at the top of the picker to have more granular control over the date range:
+
+![Analytics Date Range Picker Custom](images/analytics-date-range-picker-custom.png)
+
+On the custom picker, you can either manually enter the start and end date using the provided input fields, or use the calendar to make your selection.
+
+After either choosing a preset, or a custom date range, click the _Update_ button to modify the data in the report. Note by doing this, the date range selected is added to the address of the report URL - which allows you to bookmark, or share a report url with a specific date range to other Store admins.
```

Comment:
Could we add a paragraph here that says the date selection sticks until the user navigates away from the Analytics pages?